node-training-assignment/src/user/user-controller/create-user.js

```diff
@@ -0,0 +1,10 @@
+const userMongo = require('../user-mongo');
+
+exports.createUser = async (req, res) => {
+  try {
+    const newUser = await userMongo.createUser(req.body);
```

Comment:
Please put this line into create-user use case. Folder structure like below (user/use-case/create-user-use-case). Do it for all other files as well. Only the api related stuffs should come in controller. Like the handling of request response . What is the use case of that route (in this case to get the users from db) needs to go in the use cases